themes/src/main/resources/theme/base/login/resources/js/authChecker.js

```diff
@@ -1,5 +1,7 @@
 const CHECK_INTERVAL_MILLISECS = 2000;
+const AUTH_SESSION_INTERVAL_MILLISECS = 1000;
```

Comment:
Since the word 'interval' implies that this might repeat more than once I would recommend renaming this.

```suggestion
const AUTH_SESSION_TIMEOUT_MILLISECS = 1000;
```